project/AddMetaInfLicenseFiles.scala

```diff
@@ -17,9 +17,18 @@ import org.mdedetrich.apache.sonatype.SonatypeApachePlugin.autoImport._
  */
 object AddMetaInfLicenseFiles extends AutoPlugin {
 
+  private lazy val baseDir = LocalRootProject / baseDirectory
+
   override lazy val projectSettings = Seq(
+    apacheSonatypeLicenseFile := baseDir.value / "legal" / "StandardLicense.txt",
     apacheSonatypeDisclaimerFile := Some((LocalRootProject / baseDirectory).value / "DISCLAIMER"))
 
+  /**
+   * Settings specific for Pekko http-core subproject which require a different license file.
+   */
+  lazy val httpCoreSettings = Seq(
+    apacheSonatypeLicenseFile := baseDir.value / "LICENSE")
```

Comment:
Does that work to override the `projectSettings`? It's somewhat brittle to define the same setting twice for a project...

Reply:
This is the correct way of doing it because of how its structured for sbt-apache-sonatype and there is a test for it (see https://github.com/mdedetrich/sbt-apache-sonatype/tree/main/src/sbt-test/sbt-apache-sonatype/multi-project-different-license-test).

sbt-apache-sonatype is designed to set `apacheSonatypeLicenseFile` for every project and then a user is meant to override that value on a per project basis. Unless there is some non-determinism I am not aware of it should work fine (and can be verified with `show apacheSonatypeLicenseFile`.

Reply:
I'll merge this as is. We won't really know if this is the right solution until the Incubator PMC members look at our initial source releases.